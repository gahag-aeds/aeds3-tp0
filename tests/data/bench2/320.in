320 160
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 
Sub 68 173
Sum 200 305
Add 137 242
Sum 110 215
Add 153 258
Sum 194 299
Add 129 234
Sum 68 173
Sub 106 211
Min 21 126
Sub 34 139
Min 9 114
Sub 154 259
Max 45 150
Add 91 196
Min 82 187
Add 71 176
Min 95 200
Sub 80 185
Max 173 278
Add 211 316
Sum 10 115
Add 89 194
Min 149 254
Sub 138 243
Sum 206 311
Sub 24 129
Max 9 114
Add 151 256
Min 72 177
Add 177 282
Min 150 255
Add 57 162
Min 99 204
Add 191 296
Min 142 247
Sub 78 183
Sum 37 142
Add 209 314
Max 183 288
Sub 58 163
Max 28 133
Add 123 228
Min 211 316
Sub 4 109
Sum 213 318
Sub 10 115
Max 6 111
Add 25 130
Sum 89 194
Add 179 284
Min 168 273
Add 31 136
Sum 199 304
Sub 102 207
Max 168 273
Sub 190 295
Sum 125 230
Sub 108 213
Max 127 232
Add 197 302
Max 70 175
Sub 208 313
Min 185 290
Sub 100 205
Min 185 290
Sub 44 149
Max 177 282
Add 7 112
Min 38 143
Add 77 182
Min 210 315
Sub 212 317
Sum 199 304
Add 139 244
Max 147 252
Add 129 234
Min 148 253
Add 153 258
Min 85 190
Add 169 274
Max 49 154
Sub 38 143
Sum 131 236
Add 33 138
Max 71 176
Sub 84 189
Sum 154 259
Sub 128 233
Max 123 228
Sub 212 317
Min 42 147
Sub 124 229
Max 206 311
Sub 158 263
Sum 9 114
Sub 108 213
Min 202 307
Add 117 222
Sum 46 151
Add 25 130
Max 193 298
Sub 188 293
Max 168 273
Add 109 214
Min 112 217
Add 101 206
Max 169 274
Add 191 296
Max 185 290
Add 185 290
Min 77 182
Add 19 124
Max 155 260
Add 207 312
Min 197 302
Add 11 116
Min 8 113
Sub 68 173
Sum 70 175
Sub 62 167
Sum 66 171
Add 111 216
Sum 185 290
Add 203 308
Max 55 160
Add 125 230
Min 96 201
Sub 188 293
Min 27 132
Add 73 178
Sum 212 317
Add 151 256
Max 46 151
Sub 98 203
Max 191 296
Add 157 262
Sum 198 303
Add 77 182
Max 66 171
Sub 100 205
Min 48 153
Sub 142 247
Max 50 155
Sub 202 307
Sum 67 172
Sub 32 137
Max 144 249
Sub 74 179
Max 99 204
Sub 146 251
Sum 136 241
Sub 96 201
Max 42 147
Sub 38 143
Sum 84 189
Sub 28 133
Sum 163 268
Add 179 284
Sum 1 106
