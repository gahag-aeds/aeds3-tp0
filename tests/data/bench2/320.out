5275
5295
5568
5080
-1
-2
97
-2
2
98
5053
2
5544
99
-3
3
3
3
4918
103
96
2
5892
97
5346
4
5829
104
5676
103
99
3
3
103
-4
2
5615
103
3
-3
97
5992
102
6453
105
-4
103
4898
0
4656
103
102
1
102
103
-3
104
1
-1
4929
4830
6149
98
2
-3
5933
99
107
6052
104
-1
100
5010
106
101
5828
98
4859
6338
5065
